src/Strategy/NuGet/ProjectAssetsJson.hs

```diff
@@ -0,0 +1,93 @@
+module Strategy.NuGet.ProjectAssetsJson
+  ( discover
+  , strategy
+  , buildGraph
+  , analyze
+
+  , ProjectAssetsJson(..)
+  ) where
+
+import Prologue
+
+import qualified Data.Map.Strict as M
+import qualified Data.Text as T
+import           Polysemy
+import           Polysemy.Input
+import           Polysemy.Output
+
+import           DepTypes
+import           Discovery.Walk
+import           Effect.ReadFS
+import           Graphing (Graphing, unfold)
+import           Types
+
+discover :: Discover
+discover = Discover
+  { discoverName = "projectassetsjson"
+  , discoverFunc = discover'
+  }
+
+discover' :: Members '[Embed IO, Output ConfiguredStrategy] r => Path Abs Dir -> Sem r ()
+discover' = walk $ \_ subdirs files -> do
+  case find (\f -> fileName f == "project.assets.json") files of
+    Just file -> output (configure file)
+    Nothing -> pure ()
+
+  walkSkipNamed ["node_modules/"] subdirs
```

Comment:
do we actually want to skip `node_modules`?

Reply:
Mistake